api/Document.json

```diff
@@ -4398,9 +4398,22 @@
           "support": {
             "chrome": [
               {
-                "version_added": "113",
-                "partial_implementation": true,
-                "notes": "Only available to Google Chrome's <a href='https://developer.chrome.com/docs/privacy-sandbox/first-party-sets/'>first-party sets</a>."
+                "version_added": "119"
+              },
+              {
+                "version_added": "78",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "#storage-access-api",
+                    "value_to_set": "Enabled"
+                  }
+                ]
+              }
```

Comment:
Since the feature is enabled by default, let's remove the flag data as per https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#removal-of-irrelevant-flag-data.

Reply:
Done in https://github.com/mdn/browser-compat-data/pull/21336/commits/29c182a262804f4de7a67dfcf1b95c428abdf8d2. I also changed the `edge` implementation to "mirror" which removes it's flags too.